client/src/components/tcg-logo/tcg-logo.tsx

```diff
@@ -0,0 +1,15 @@
+import css from './tcg-logo.module.css'
```

Comment:
Please add a `index.ts` file with `export {default} from './tcg-logo'`.
Then change your imports to
`import TcgLogo from 'components/tcg-logo'`

Same for link-container.